lightdash/lightdash
# fix: catalog edges migrations

Closes: #12708 

### Description:

- Comments out code from partially failing migrations
- Creates migration to remove changes from previous failing migrations where they might've been applied
- Creates migration to add new changes referencing `uuid` and not creating `unique` constraint on `catalog_search`

**Steps to test**

We need to run the following steps against 3 git refs:
- 551f608eb - commit before any of the failing migrations
- d33ad3571 - commit after failing migrations but before https://github.com/lightdash/lightdash/pull/12702
- `main` - current code with all the failing migrations + fix attempt

1. Checkout to the selected commit
2. Reset db by running `./scripts/reset-db.sh`
3. Checkout current branch
4. Run migrate
5. Refresh dbt
6. Run rollbacks to make sure they aren't broken

### Reviewer actions
